### ISSUE TITLE
UI is broken on dark themes.

### ISSUE THREAD

**Issue by SkyzohKey** [ryonakano/Spreadsheet#10]
![Spreadsheet on black theme](https://0x0.st/37n.png)

I'm using Adapta theme under SolusOS.

**Comment by Bat41** [comment]
Yes, I have hardcoded colors in the `Sheet` widget. But I don't know how to get the colors from the theme, so I just took color that worked well in elementary (since the app is designed for this distribution). But if you know how to get the selection and background color from the GTK+ theme, tell me, i'll change it. :slightly_smiling_face:

**Comment by SkyzohKey** [comment]
Mh, `Gtk.StyleContext.get_background_color ()`, `Gtk.StyleContext.get_border_color ()` and `Gtk.StyleContext.get_color ()` iirc. :p

**Comment by Bat41** [comment]
Yes, I tried, but I don't give me the colors I want: selection color is gray, and background too. I think i'll look in the `Gtk.Entry` source code to see how they find their colors (because I would like to have the same).

**Comment by SkyzohKey** [comment]
You tried by getting the stylecontext of the main window ?

**Comment by SkyzohKey** [comment]
What about adding the .entry class to the cells ?

**Comment by Bat41** [comment]
That may work, I'll try.

**Comment by SkyzohKey** [comment]
The GtkEntry code to get the placeholder color is:
```c
static void
gtk_entry_get_placeholder_text_color (GtkEntry   *entry,
                                      PangoColor *color)
{
  GtkWidget *widget = GTK_WIDGET (entry);
  GtkStyleContext *context;
  GdkRGBA fg = { 0.5, 0.5, 0.5 };

  context = gtk_widget_get_style_context (widget);
  gtk_style_context_lookup_color (context, "placeholder_text_color", &fg);

  color->red = CLAMP (fg.red * 65535. + 0.5, 0, 65535);
  color->green = CLAMP (fg.green * 65535. + 0.5, 0, 65535);
  color->blue = CLAMP (fg.blue * 65535. + 0.5, 0, 65535);
}
```

So it has to do with the StyleContext ^^

**Comment by Bat41** [comment]
OK, so we can get a white background using the style context of the widget if we add the `entry` class! I guess it will also work with the other colors, so I'll change that.

**Comment by SkyzohKey** [comment]
Now thing is to find the list of avilable colors names/ids :x

**Comment by SkyzohKey** [comment]
Oh, nice ! :)

**Comment by SkyzohKey** [comment]
Ok so "placeholder_text_color" is a value defined in gtk.css

**Comment by ryonakano** [comment]
Should be fixed in #13
